test/integration/glue_test.py

```diff
@@ -40,97 +42,110 @@ def glue_test_dfs(
     glue_test_dfs = {
         "plants_eia_pudl_db": pudl_out.plants_eia860(),
         "util_ids_ferc1_raw_xbrl": get_util_ids_ferc1_raw_xbrl(ferc1_engine_xbrl),
-        "util_ids_ferc1_raw_dbf": get_utils_ferc1_raw_dbf(ferc1_engine_dbf),
-        "plants_ferc1_raw": get_raw_plants_ferc1(
+        "util_ids_ferc1_raw_dbf": get_util_ids_ferc1_raw_dbf(ferc1_engine_dbf),
+        "plants_ferc1_raw": get_plants_ferc1_raw(
             pudl_settings=pudl_settings_fixture,
             years=DataSource.from_id("ferc1").working_partitions["years"],
```

Comment:
Should this always be all years, even in the tests? Or should it be the years that were read in from the settings file?

Reply:
definitely should come from the settings. good catch.